Phaethon/InternalApi/Migrations/201812050952594_removed DispalyNames from job and customer tables.cs

```diff
@@ -0,0 +1,16 @@
+namespace InternalApi.Migrations
+{
+    using System;
+    using System.Data.Entity.Migrations;
+    
+    public partial class removedDispalyNamesfromjobandcustomertables : DbMigration
```

Comment:
empty migration, no need to have it

Reply:
Read previous message, it doesn't allow for it.

Reply:
answered